src/AccessibilityInsights.SharedUx/Controls/TestTabs/TabStopControl.xaml.cs

```diff
@@ -448,6 +444,17 @@ private void OnFocusChanged(EventMessage message)
                     message.Dispose();
                 }
             }
+            else if (IsMessageRegisterSuccess(message))
```

Comment:
I wonder if two changes would help with readability here:
- move the `IsMessageRegisterSuccess` case to the top of the function since it occurs first time-wise
- change the name of the method to `eventReceived` or `focusEventReceived` so it's a bit more clear that other messages can appear

I'm not sure how common our usage pattern is (triggering an event after registration success but before first focus event), but future thinking in Axe.Windows might be helpful to improve the interface for clients (so they don't need to know about the details of `IsMessageRegisterSuccess` as you mention)

Reply:
Agree with rename. But `IsMessageRegisterSuccess` is the significantly less commonly expected branch path and has to do string comparisons on every event property pair, so I'd prefer that we don't call it more than is necessary. Do you feel like its later location is a significant hindrance to understanding the function?

Reply:
I was mainly confused that this method can be called with non-focus changed events - a text comment would work too if we want to maintain the structure as-is. The string comparisons will likely be short-circuited out, if that helps at all

Reply:
Hopefully it is now more readable